### PR TITLE
Upgrade to hyper 0.13.0-alpha.1

### PR DESCRIPTION
**Pull request by jhpratt**
Pinning to avoid any unexpected breakage

### CODE REVIEW

File: core/lib/src/rocket.rs [hunk 1]
```diff
@@ -151,7 +151,7 @@ impl Rocket {
                     let mut stream = body.into_chunk_stream(4096);
                     while let Some(next) = stream.next().await {
                         futures::future::poll_fn(|cx| sender.poll_ready(cx)).await.expect("TODO.async client gone?");
-                        sender.send_data(next?).expect("send chunk");
+                        sender.send_data(next?).await.expect("send chunk");
```
Comment (jebrosen) [inline core/lib/src/rocket.rs]: This was renamed/refactored in hyper. We should remove the `poll_ready` line above and move its TODO to this line.

File: core/lib/src/rocket.rs [hunk 2]
```diff
@@ -163,7 +163,7 @@ impl Rocket {
                     let mut stream = body.into_chunk_stream(chunk_size.try_into().expect("u64 -> usize overflow"));
                     while let Some(next) = stream.next().await {
                         futures::future::poll_fn(|cx| sender.poll_ready(cx)).await.expect("TODO.async client gone?");
-                        sender.send_data(next?).expect("send chunk");
+                        sender.send_data(next?).await.expect("send chunk");
```
Comment (jebrosen) [inline core/lib/src/rocket.rs]: Same as above.

File: core/lib/src/rocket.rs (hunk 1 above)
Comment (jhpratt) [reply core/lib/src/rocket.rs]: Done.

**Comment by jebrosen** [comment]
Great! Merged as 5e5043c.

**Pull request closed without merging**